### PR TITLE
feat: add TemplateEngine for KafkaConnectionContext

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://gravitee.atlassian.net/browse/APIM-7719

**Description**

Juste add String used by this PR https://github.com/gravitee-io/gravitee-reactor-native-kafka/pull/41

Similar to KafkaExecutionContext but for Connection

**Additional context**

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `3.9.0-APIM-7719-template-engine-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/gateway/gravitee-gateway-api/3.9.0-APIM-7719-template-engine-SNAPSHOT/gravitee-gateway-api-3.9.0-APIM-7719-template-engine-SNAPSHOT.zip)

### CODE REVIEW

File: src/main/java/io/gravitee/gateway/reactive/api/context/kafka/KafkaConnectionContext.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
  * @author GraviteeSource Team
  */
 public interface KafkaConnectionContext extends NativeExecutionContext {
+    String TEMPLATE_ATTRIBUTE_CONTEXT = "context";
```
Comment (jhaeyaert) [inline src/main/java/io/gravitee/gateway/reactive/api/context/kafka/KafkaConnectionContext.java:29]: I would rather move HttpPlainExecutionContext.TEMPLATE_ATTRIBUTE_CONTEXT to BaseExecutionContext so it is inherited by all the sub interfaces and not duplicated. wdyt?

**Review by jhaeyaert — CHANGES_REQUESTED** [review]

File: src/main/java/io/gravitee/gateway/reactive/api/context/kafka/KafkaConnectionContext.java (hunk 1 above)
Comment (ThibaudAV) [reply src/main/java/io/gravitee/gateway/reactive/api/context/kafka/KafkaConnectionContext.java:29]: yes 🚀

**Review by jhaeyaert — APPROVED** [review]

**Pull request merged**